paasta_tools/deployd/workers.py

```diff
@@ -68,14 +71,18 @@ def setup_timers(self, service_instance):
         )
 
     def run(self):
+        """Takes things from the to_bounce_now queue, processes them, then
+        might put them on the bounce_later queue for future processing"""
         self.log.info(f"{self.name} starting up")
         while True:
             service_instance = self.instances_to_bounce_now.get()
             try:
                 bounce_again_in_seconds, return_code, bounce_timers = self.process_service_instance(service_instance)
             except Exception as e:
-                self.log.error("Worker failed to process service instance and will retry. "
-                               "Caused by exception: {}".format(e))
+                self.log.error(
+                    f"{self.name} Worker failed to process service instance and will retry. "
+                    f"Caused by exception: {format(e)}",
```

Comment:
is `format(e)` what you want? I don't actually know what it does...

Reply:
It is. Otherwise we get pages and pages of stacktrace